GPU-SUPPORT.md

```diff
@@ -0,0 +1,19 @@
+# GPU Support in Livepeer
+
+Livepeer enables node operators to transcode video on GPUs while concurrently mining cryptocurrencies. As there is a very wide range of GPU hardware out there, this document aims to crowdsource a list of specific GPU models that have been tested and verified to work on Livepeer. If you've tested an additional model, please submit an update to this document.
+
+**Overview**
+
+* Livepeer supports transcoding on NVidia GPUs with NVENC/NVDEC. Any GPU [listed here](https://developer.nvidia.com/video-encode-and-decode-gpu-support-matrix-new) with those chips should theoretically work. Note that different models have different driver versions and may be subject to session limits.
```

Comment:
Nit: I always thought it was Nvidia as opposed to NVidia?

Reply:
Looks like everyone in their own marketing it is NVIDIA. Will change.